sysl2/sysl/cmd_repl.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os"
+
+	"github.com/anz-bank/sysl/sysl2/sysl/eval"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type replCmd struct{}
+
+func (p *replCmd) Name() string            { return "repl" }
+func (p *replCmd) RequireSyslModule() bool { return false }
+
+func (p *replCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
+	cmd := app.Command(p.Name(), "Enter a sysl REPL")
+	return cmd
+}
+
+func (p *replCmd) Execute(args ExecuteArgs) error {
+	s := &eval.Scope{}
+	repl := eval.NewREPL(os.Stdin, os.Stdout)
+	for {
```

Comment:
Its an infinite loop that breaks on error, what if there is no error?

Reply:
expected use is user presses q to exit